spec/lib/connection_spec.rb

```diff
@@ -147,7 +147,7 @@
       expect(auth_header == "APIAuth id123:TQiQIW6vVaDC5jvh99uTNkxIg6Q=" || auth_header == "APIAuth id123:PMWBThkB8vKbvUccHvoqu9G3eVk=").to be_truthy
     end
 
-    if Gem.loaded_specs["api-auth"].version.to_s >= "2.0.0"
+    if  Gem.loaded_specs["api-auth"].present? && Gem.loaded_specs["api-auth"].version.to_s >= "2.0.0"
```

Comment:
Hard to say, but on my machine I had 
`undefined method `version' for nil:NilClass (NoMethodError)`
when running Specs, as 'api-auth' key wasn't loaded into loaded_specs. This is my non-invasive way of fixing that.